### PR TITLE
Duplicate services/ URLs under /api/v5/services/

### PR DESCRIPTION
**Pull request by diox**
This makes it easier to reach those views for testing purposes, enabling us to reach `client_info/` from `services.a.m.o.` domain (which only accepts specific URL patterns like `/api/`) or test the various 40x/500 handlers from an API perspective.

Supports https://github.com/mozilla/addons-server/issues/21628 (will help test that my implementation for it is correct)

This enables:
- http://olympia.test/api/v5/services/500
- http://olympia.test/api/v5/services/404
- http://olympia.test/api/v5/services/403
- http://olympia.test/api/v5/services/client_info (if `settings.ENV` is `dev`)

### CODE REVIEW

File: src/olympia/amo/urls.py [hunk 1]
```diff
@@ -17,6 +17,11 @@
 
 api_patterns = [
     re_path(r'^site/$', views.SiteStatusView.as_view(), name='amo-site-status'),
+    # This duplicates services patterns under /api/vX/services/, which is
+    # useful to reach those services from an API request, to make them work
+    # with services.a.m.o. or test their behavior with request.is_api being
+    # True.
+    re_path(r'^services/', include(services_patterns)),
```
Comment (eviljeff) [inline src/olympia/amo/urls.py:24]: some of the patterns - the 403, 403, 500 views - are not JSON so should they really be exposed under /api?  (😬)

**Review by eviljeff — COMMENTED** [review]
can haz docs

File: src/olympia/amo/urls.py (hunk 1 above)
Comment (diox) [reply src/olympia/amo/urls.py:24]: `404` and `500` are. Only `403` is not. I could implement a quick one.

Comment (diox) [reply src/olympia/amo/urls.py]: 44ce9f42fbd206bdf7341d8b1d744e41e17b573f adds docs, limits to the views we care about, adds 403 API support, and tests.

**Review by eviljeff — APPROVED** [review]

**Pull request merged**